Products/PrintingMailHost/Patch.py

```diff
@@ -1,14 +1,12 @@
-import email.Parser
-try:
-    from email.message import Message
-except ImportError:
-    from email import Message
-from base64 import decodestring
-
+# -*- coding: utf-8 -*-
 from AccessControl import ClassSecurityInfo
-from Products.PrintingMailHost import LOG, FIXED_ADDRESS
+from base64 import decodestring
+from email.message import Message
```

Comment:
This `email.Message` import will fail on Python 2.4. That is what the `except ImportError` was for.
This is fine of course, but it deserves a note in the changelog, and the Python 2.4 and Plone 3.3 classifiers should be removed.

Reply:
👍